src/diffusers/pipelines/flux/pipeline_flux.py

```diff
@@ -668,6 +705,11 @@ def __call__(
             latents,
         )
 
+        if not self._are_latents_packed(latents):
+            raise ValueError(
+                "Latents are not in the correct packed format. Please use `_pack_latents` to prepare them."
+            )
```

Comment:
This can be removed too since essentially `check_inputs()` should catch these.

Reply:
Yes, the `check_inputs` function should catch these, so this can be removed. Then there is no need for the `_are_latents_packed` method, so that can also be removed.